src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_horizontal_layout.html.twig

```diff
@@ -49,6 +49,7 @@ col-sm-2
             <div class="{{ block('form_group_class') }}">
                 {{- form_widget(form, widget_attr) -}}
                 {{- form_help(form) -}}
+                {{- form_errors(form) -}}
```

Comment:
Is this really necessary as the errors should be rendered when the label is rendered?